frontend/components/Nolla/layout.tsx

```diff
@@ -150,7 +150,8 @@ function NollaLayout({
           background: 'hsl(317, 82%, 56%)',
           color: 'white',
         },
-        minHeight: '100vh',
+        height: '100vh',
+        overflowY: 'auto',
```

Comment:
Don't you think we might as well make the container surrounding the children the scroll container? As you pointed out yesterday the scrollbar should be below the AppBar.